Package.swift

```diff
@@ -17,12 +17,12 @@ let package = Package(
         ),
     ],
     dependencies: [
-        .package(url: "https://github.com/SnapKit/SnapKit.git", .upToNextMajor(from: "5.6.0")),
+        .package(url: "https://github.com/SnapKit/SnapKit.git", from: "5.6.0"),
         .package(url: "https://github.com/thumbtack/TTCalendarPicker.git", from: "0.2.0"),
         .package(url: "https://github.com/pointfreeco/swift-snapshot-testing.git", from: "1.11.0"),
         .package(url: "https://github.com/thumbtack/thumbprint-tokens.git", from: "13.0.1"),
-        .package(url: "https://github.com/realm/SwiftLint.git", from: "0.51.0"),
-        .package(url: "https://github.com/nicklockwood/SwiftFormat", from: "0.50.4"),
+        .package(url: "https://github.com/lukepistrol/SwiftLintPlugin", from: "0.2.6"),
+        .package(url: "https://github.com/nicklockwood/SwiftFormat", from: "0.51.9"),
```

Comment:
Did you mean to bump this to 51.9 , i see you changing it *back* to 51.8 in other PRs ?

Reply:
yes, I will bump to 0.51.9 in here and I will do the same in the main app too